test/Faker/Provider/ru_RU/PersonTest.php

```diff
@@ -29,4 +30,135 @@ protected function getProviders(): iterable
     {
         yield new Person($this->faker);
     }
+
+    /**
+     * @dataProvider dataLastNameMale
+     */
+    public function testLastNameMale(int $seed, string $expected): void
+    {
+        $generator = new Generator();
+        $generator->seed($seed);
+
+        $provider = new Person($generator);
+
+        self::assertSame($expected, $provider->lastNameMale());
+    }
+
+    public function dataLastNameMale(): iterable
+    {
+        yield 'seed: 4' => [
+            4,
+            'Морозов',
+        ];
+
+        yield 'seed: 8' => [
+            8,
+            'Гусев',
+        ];
+
+        yield 'seed: 15' => [
+            15,
+            'Алексеев',
+        ];
+
+        yield 'seed: 16' => [
+            16,
+            'Фадеев',
+        ];
+
+        yield 'seed: 23' => [
+            23,
+            'Воронов',
+        ];
+
+        yield 'seed: 42' => [
+            42,
+            'Горбачёв',
+        ];
+    }
+
+    /**
+     * @dataProvider dataLastNameFemale
+     */
+    public function testLastNameFemale(int $seed, string $expected): void
+    {
+        $generator = new Generator();
+        $generator->seed($seed);
+
+        $provider = new Person($generator);
+
+        self::assertSame($expected, $provider->lastNameFemale());
+    }
+
+    public function dataLastNameFemale(): iterable
+    {
+        yield 'seed: 4' => [
+            4,
+            'Морозова',
+        ];
+
+        yield 'seed: 8' => [
+            8,
+            'Гусева',
+        ];
+
+        yield 'seed: 15' => [
+            15,
+            'Алексеева',
+        ];
+
+        yield 'seed: 16' => [
+            16,
+            'Фадеева',
+        ];
+
+        yield 'seed: 23' => [
+            23,
+            'Воронова',
+        ];
+
+        yield 'seed: 42' => [
+            42,
+            'Горбачёва',
+        ];
+    }
+
+    /**
+     * @dataProvider dataIssue832
+     */
+    public function testIssue832(int $seed, string $expected): void
```

Comment:
Please describe a valid test scenario instead of an issue number. Nobody will understand this in the future

Reply:
done